fcrepo-configs/src/main/resources/config/infinispan/file/infinispan.xml

```diff
@@ -1,39 +1,36 @@
 <infinispan xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:schemaLocation="urn:infinispan:config:6.0 http://www.infinispan.org/schemas/infinispan-config-6.0.xsd"
-            xmlns="urn:infinispan:config:6.0">
+            xsi:schemaLocation="urn:infinispan:config:7.2 http://www.infinispan.org/schemas/infinispan-config-7.2.xsd"
+            xmlns="urn:infinispan:config:7.2">
 
-  <global>
-    <globalJmxStatistics enabled="true" allowDuplicateDomains="true"/>
-  </global>
+  <cache-container default-cache="FedoraRepository">
 
-  <default>
-  </default>
+    <jmx duplicate-domains="true"/>
 
+    <local-cache name="FedoraRepository" statistics="true">
 
-  <namedCache name="FedoraRepository">
+      <transaction transaction-manager-lookup="org.infinispan.transaction.lookup.GenericTransactionManagerLookup"
+                   mode="NON_XA"
+                   locking="PESSIMISTIC"/>
 
-    <eviction maxEntries="500"  strategy="LIRS" threadPolicy="DEFAULT"/>
+      <eviction max-entries="500" strategy="LIRS" thread-policy="DEFAULT"/>
 
-    <transaction
-            transactionManagerLookupClass="org.infinispan.transaction.lookup.GenericTransactionManagerLookup"
-            transactionMode="TRANSACTIONAL" lockingMode="PESSIMISTIC"/>
-    <!--
-        Define the cache loaders (i.e., cache stores). Passivation is false because we want *all*
-        data to be persisted, not just what doesn't fit into memory. Shared is false because there
-        are no other caches sharing this file store. We set preload to false for lazy loading;
-        may be improved by preloading and configuring eviction.
+      <!--
+          Define the cache loaders (i.e., cache stores). Passivation is false because we want *all*
+          data to be persisted, not just what doesn't fit into memory. Shared is false because there
+          are no other caches sharing this file store. We set preload to false for lazy loading;
+          may be improved by preloading and configuring eviction.
 
-        We can have multiple cache loaders, which get chained. But we'll define just one.
-     -->
+          We can have multiple cache loaders, which get chained. But we'll define just one.
+       -->
+      <persistence passivation="false">
+        <file-store shared="false"
+                    preload="false"
+                    fetch-state="false"
```

Comment:
Is there a reason this is different from fetchPersistentState in the old config?  Reading the user guide explanation of the fetch-state option, it sounds like we want fetch-state=true: http://infinispan.org/docs/7.2.x/user_guide/user_guide.html#_configuration_2